test/dotnet-pack.Tests/PackTests.cs

```diff
@@ -95,6 +95,29 @@ public void HasBuildOutputWhenUsingBuildBasePath()
             zip.Entries.Should().Contain(e => e.FullName == "lib/netstandardapp1.5/TestLibraryWithConfiguration.dll");
         }
 
+        [Fact]
+        public void PackAddsCorrectFilesForProjectsWithOutputNameSpecified()
```

Comment:
Just noticed `HasBuildOutputWhenUsingBuildBasePath` is testing the contents of the package.

My concern was just that I didn't see the other tests testing the actual contents. 

:+1: